src/guide/essentials/reactivity-fundamentals.md

```diff
@@ -67,98 +67,153 @@ When you access `this.someObject` after assigning it, the value is a reactive pr
 
 <div class="composition-api">
 
-We can create a reactive object or array with the [`reactive()`](/api/reactivity-core#reactive) function:
+## Declaring Reactive State \** {#declaring-reactive-state-1}
+
+### `ref()` \** {#ref}
+
+In Composition API, the recommended way to declare reactive state is using the [`ref()`](/api/reactivity-core#ref) function:
 
 ```js
-import { reactive } from 'vue'
+import { ref } from 'vue'
 
-const state = reactive({ count: 0 })
+const count = ref(0)
 ```
 
-Reactive objects are [JavaScript Proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) and behave just like normal objects. The difference is that Vue is able to track the property access and mutations of a reactive object. If you are curious about the details, we explain how Vue's reactivity system works in [Reactivity in Depth](/guide/extras/reactivity-in-depth) - but we recommend reading it after you have finished the main guide.
+`ref()` takes the argument and returns it wrapped within a ref object with a `.value` property:
 
-See also: [Typing Reactive](/guide/typescript/composition-api#typing-reactive) <sup class="vt-badge ts" />
+```js
+const count = ref(0)
 
-To use reactive state in a component's template, declare and return them from a component's `setup()` function:
+console.log(count) // { value: 0 }
+console.log(count.value) // 0
+
+count.value++
+console.log(count.value) // 1
+```
+
+> See also: [Typing Refs](/guide/typescript/composition-api#typing-ref) <sup class="vt-badge ts" />
+
+To access refs in a component's template, declare and return them from a component's `setup()` function:
 
 ```js{5,9-11}
-import { reactive } from 'vue'
+import { ref } from 'vue'
 
 export default {
-  // `setup` is a special hook dedicated for the composition API.
+  // `setup` is a special hook dedicated for the Composition API.
   setup() {
-    const state = reactive({ count: 0 })
+    const count = ref(0)
 
-    // expose the state to the template
+    // expose the ref to the template
     return {
-      state
+      ref
     }
   }
 }
 ```
 
 ```vue-html
-<div>{{ state.count }}</div>
+<div>{{ count }}</div>
 ```
 
-Similarly, we can declare functions that mutate reactive state in the same scope and expose them as methods alongside the state:
+Notice that we did **not** need to append `.value` when using the ref in the template. For convenience, refs are automatically unwrapped when used inside templates. You can also mutate a ref directly in event handlers:
 
```

Comment:
Maybe link to template unwrapping caveat here? Not 100% confident but nested ref not being unwrapped might be surprising for user if we start from teaching `ref` 👀 

Something like:

```
Note: A `ref` nested inside object will not be unwrapped. Learn more at...
```